templates/common-partials/script/pagination.hbs

```diff
@@ -0,0 +1,19 @@
+ANSWERS.addComponent("Pagination", Object.assign({}, {
+  container: "#js-answersPagination",
+  onPaginate: (newPageNumber, oldPageNumber, totalPages) => {
+    if (window.parentIFrame) {
+      const paginateMessage = { action: 'paginate' };
+      window.parentIFrame.sendMessage(JSON.stringify(paginateMessage));
+    } else {
+      document.documentElement.scrollTop = 0;
+      document.body.scrollTop = 0; // Safari
+    }
+    const resultsColumn = document.querySelector(".js-locator-resultsWrapper");
```

Comment:
I did a little more thinking and I think it might be better to have the vertical-full-page-map not use the common partial, at least for now, and we can remove lines 11-14 from the common partial. It's a little spaghetti to have the shared code have logic that is specific to a single page template. In the future we probably want some way to insert page template specific code into the common partial, but I think that's out of scope for this PR.